package.json

```diff
@@ -18,7 +18,7 @@
     "test": "run-s -c lint lint:py format:test test-unit test:legacy test:intg test:pyimport",
     "test:legacy": "pytest tests/test_integration*.py",
     "test:intg": "pytest --nopercyfinalize --headless tests/integration",
-    "test:pyimport": "python -m unittest tests/test_dash_import.py",
+    "test:pyimport": "echo $(python -V 2>&1) | grep 3. && python -m unittest tests/test_dash_import.py",
```

Comment:
Worries me a little that this is completely silent about skipping this test - can we have it show a message about what it did at least? something like `|| echo "skipping test_dash_import in Python 2.x"`?

Reply:
oh you moved this into the `circle.yml` - but I think the same comment applies there.

Reply:
yes, the NPM version does not work. I remember this test was ignored for quite a long time, and I think the tests in Python3 should be enough to cover this https://github.com/plotly/dash-core-components/blob/474f196d285e83cbe750bbbab998b220a0d61ee2/dash_core_components/__init__.py#L18-L23